components/actionbutton/stories/template.js

```diff
@@ -128,3 +128,75 @@ export const Template = ({
 		</button>
 	`;
 };
+
+export const ActionButtonsWithIconOptions = ({
+	...args
+}, context ) => Container({
+	withBorder: false,
+	direction: "row",
+	wrapperStyles: {
+		columnGap: "12px",
+	},
+	content: html`
+    ${Template({
+      ...args,
+    }, context )}
+    ${Template({
+      ...args,
+    }, context )}
```

Comment:
The first two are the same here. I think the second is supposed to exclude the icon?